### PR TITLE
Add ice nucleation parameters

### PR DESCRIPTION
**Pull request by trontrytel**
@amylu00 - could you take a look and add the DOI numbers, units and maybe improve the descriptions and/or parameter names?

### CODE REVIEW

File: src/parameters.toml [hunk 1]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+alias = "Koop2000_max_delta_aw"
+value = 0.34
+type = "float"
+description = "A coefficient from Koop 2000 TODO - add DOI number, units and description"
+
+[Koop2000_J_hom_coeff1]
+alias = "Koop2000_J_hom_c1"
+value = -906.7
+type = "float"
+description = "A coefficient from Koop 2000 TODO - add DOI number, units and description"
```
Comment (amylu00) [inline src/parameters.toml:1245]:
```suggestion
description = "Coefficient for calculating J_hom parameterized by Koop 2000 [-]. From Koop 2000. DOI: 10.1038/35020537."
```

File: src/parameters.toml [hunk 2]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+
+[Koop2000_J_hom_coeff2]
+alias = "Koop2000_J_hom_c2"
+value = 8502
+type = "float"
+description = "A coefficient from Koop 2000 TODO - add DOI number, units and description"
```
Comment (amylu00) [inline src/parameters.toml:1251]:
```suggestion
description = "Coefficient for calculating J_hom parameterized by Koop 2000 [-]. From Koop 2000. DOI: 10.1038/35020537."
```

File: src/parameters.toml [hunk 3]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+
+[Koop2000_J_hom_coeff3]
+alias = "Koop2000_J_hom_c3"
+value = 26924
+type = "float"
+description = "A coefficient from Koop 2000 TODO - add DOI number, units and description"
```
Comment (amylu00) [inline src/parameters.toml:1257]:
```suggestion
description = "Coefficient for calculating J_hom parameterized by Koop 2000 [-]. From Koop 2000. DOI: 10.1038/35020537."
```

File: src/parameters.toml [hunk 4]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+
+[Koop2000_J_hom_coeff4]
+alias = "Koop2000_J_hom_c4"
+value = 29180
+type = "float"
+description = "A coefficient from Koop 2000 TODO - add DOI number, units and description"
```
Comment (amylu00) [inline src/parameters.toml:1263]:
```suggestion
description = "Coefficient for calculating J_hom parameterized by Koop 2000 [-]. From Koop 2000. DOI: 10.1038/35020537."
```

File: src/parameters.toml [hunk 5]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Desert Dust TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_c_DesterDust]
+alias = "J_het_c_KA2013_DesertDust"
+value = -1.35
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Desert Dust TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_m_Kaolinite]
```
Comment (amylu00) [inline src/parameters.toml:1205]:
```suggestion
[KnopfAlpert2013_J_ABIFM_m_Kaolinite]
```

File: src/parameters.toml [hunk 6]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
 description = "a for temperatures < T_thr for Desert Dust [-]. From Mohler et al, 2006. DOI: 10.5194/acp-6-3007-2006."
 
+[KnopfAlpert2013_J_het_m_DesterDust]
+alias = "J_het_m_KA2013_DesertDust"
+value = 22.62
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Desert Dust TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_c_DesterDust]
+alias = "J_het_c_KA2013_DesertDust"
```
Comment (amylu00) [inline src/parameters.toml:1200]:
```suggestion
alias = "J_ABIFM_c_KA2013_DesertDust"
```

File: src/parameters.toml [hunk 7]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
 type = "float"
 description = "a for temperatures < T_thr for Desert Dust [-]. From Mohler et al, 2006. DOI: 10.5194/acp-6-3007-2006."
 
+[KnopfAlpert2013_J_het_m_DesterDust]
+alias = "J_het_m_KA2013_DesertDust"
```
Comment (amylu00) [inline src/parameters.toml:1194]:
```suggestion
alias = "J_ABIFM_m_KA2013_DesertDust"
```

File: src/parameters.toml [hunk 8]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+description = "A coefficient from Knopf and Alpert 2013 for Desert Dust TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_c_DesterDust]
+alias = "J_het_c_KA2013_DesertDust"
+value = -1.35
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Desert Dust TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_m_Kaolinite]
+alias = "J_het_m_KA2013_Kaolinite"
```
Comment (amylu00) [inline src/parameters.toml:1206]:
```suggestion
alias = "J_ABIFM_m_KA2013_Kaolinite"
```

File: src/parameters.toml [hunk 9]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+value = 54.58834
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Kaolinite TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_c_Kaolinite]
```
Comment (amylu00) [inline src/parameters.toml:1211]:
```suggestion
[KnopfAlpert2013_J_ABIFM_c_Kaolinite]
```

File: src/parameters.toml [hunk 10]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+alias = "J_het_c_KA2013_Kaolinite"
```
Comment (amylu00) [inline src/parameters.toml:1212]:
```suggestion
alias = "J_ABIFM_c_KA2013_Kaolinite"
```

File: src/parameters.toml [hunk 11]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+value = -10.54758
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Kaolinite TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_m_Illite]
```
Comment (amylu00) [inline src/parameters.toml:1217]:
```suggestion
[KnopfAlpert2013_J_ABIFM_m_Illite]
```

File: src/parameters.toml [hunk 12]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+alias = "J_het_m_KA2013_Illite"
```
Comment (amylu00) [inline src/parameters.toml:1218]:
```suggestion
alias = "J_ABIFM_m_KA2013_Illite"
```

File: src/parameters.toml [hunk 13]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+value = 54.48075
+type = "float"
+description = "A coefficient from Knopf and Alpert 2013 for Illite TODO - add DOI number and units "
+
+[KnopfAlpert2013_J_het_c_Illite]
```
Comment (amylu00) [inline src/parameters.toml:1223]:
```suggestion
[KnopfAlpert2013_J_ABIFM_c_Illite]
```

File: src/parameters.toml [hunk 14]
```diff
@@ -1190,6 +1190,78 @@ value = 2.35
...
+alias = "J_het_c_KA2013_Illite"
```
Comment (amylu00) [inline src/parameters.toml:1224]:
```suggestion
alias = "J_ABIFM_c_KA2013_Illite"
```

**Review by amylu00 — COMMENTED** [review]
Sorry Anna, I wasn't able to pull this branch onto my local rep so I just changed the code through the review (my terminal said this branch doesn't exist?). I think GitHub should let you accept changes and it'll automatically commit though. If not, I can try to find a way to just clone this branch or something and make the changes that way

**Comment by amylu00** [comment]
Figured out how to commit them all remotely, there's a lot of commits to squash and rebase now, sorry!

**Comment by trontrytel** [comment]
Thank you! I wonder why you couldn't pull to your local. Lets try to fix it next time. Nothing should prevent you from adding more parameters ;)

**Pull request merged**